integration-cli/requirements_test.go

```diff
@@ -69,6 +71,17 @@ func IsAmd64() bool {
 	return os.Getenv("DOCKER_ENGINE_GOARCH") == "amd64"
 }
 
+func NotWindowsRS5Plus() bool {
```

Comment:
Might be good to copy the comment you put  above `TestRestartStoppedContainer` here ("Certain tests here have testRequires(c, NotWindowsRS5Plus)....")

Reply:
Sure, done. Think I have it now, had to disable another test. Got one clean run with a successful cleanup locally. Let's see how this goes on the production servers.